vue-frontend/src/components/home-new/CTASection.vue

```diff
@@ -1,78 +1,112 @@
+          How do you know we’re a good fit for you?
+        </div>
+        <div class="v2-title-3-text mt-4">Let’s have a short chat.</div>
+        <div class="communication">
+          <div class="item px-4 py-3">
+            <div class="v2-normal-3-text">I love talking.</div>
+            <router-link class="gradient-btn cta-button" :to="contactURL">
+              <font-awesome-icon
+                class="fa v2-title-3-text"
+                icon="phone"
+                aria-hidden="true"
+              />
+              <span class="v2-normal-3-text">Schedule a call</span>
+            </router-link>
+          </div>
+          <div class="item px-4 py-3">
+            <div class="v2-normal-3-text">I prefer mail.</div>
+            <router-link class="gradient-btn cta-button" :to="contactURL">
+              <font-awesome-icon
+                class="fa v2-title-3-text"
+                icon="paper-plane"
+                aria-hidden="true"
+              />
+              <span class="v2-normal-3-text">Drop a mail</span>
+            </router-link>
+          </div>
         </div>
-        <div class="item px-4 py-3">
-          <div class="v2-normal-3-text">I prefer mail.</div>
-          <router-link class="gradient-btn cta-button" :to="contactURL">
-            <font-awesome-icon
-              class="fa v2-title-3-text"
-              icon="paper-plane"
-              aria-hidden="true"
-            />
-            <span class="v2-normal-3-text">Drop a mail</span>
-          </router-link>
+        <div
+          class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        >
+          Four reasons why do we do this call?
         </div>
+        <ul class="v2-normal-2-text chat-list text-left mt-4">
+          <li>
+            <span class="bullet"></span>
+            <div>It gives an opportunity to learn more about each other.</div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              We want to give you a chance to put your idea into action. You
+              might have faced this situation before where you wish you had
+              someone to help you.
+            </div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              Sometimes, you want to check the feasibility of your idea. We will
+              give you honest feedback based on our industry experience.
+            </div>
+          </li>
+          <li>
+            <span class="bullet"></span>
+            <div>
+              We offer 100% Money-back guarantee. so, there’s no risk to you.
+            </div>
+          </li>
+        </ul>
       </div>
       <div
-        class="v2-title-3-text text-left v2-canopas-gradient-text reason-text mt-5"
+        class="tw-text-center ... tw-pt-16 tw-pb-32 xl:tw-pb-1"
+        :class="[
+          currentRoutePath == '/portfolio/' + id ? '' : 'hide-copyright-text',
```

Comment:
Hide that whole `copyright div` not only text 
see the difference 

1. after add copyright ( CTA section in home page )
![Screenshot from 2022-06-30 16-36-42](https://user-images.githubusercontent.com/94360312/176662530-1745bf94-b99d-4da6-a58e-1cb3ca986f53.png)

2. Before CTA section in home page 
![Screenshot from 2022-06-30 16-36-20](https://user-images.githubusercontent.com/94360312/176662912-ea827cb9-93a8-45c5-bf96-1ebf649ef277.png)